app/views/sites/visual_designs/_fonts_and_colors.html.haml

```diff
@@ -10,4 +10,12 @@
     options_for_select(Affiliate::USWDS_FONTS.index_by { |font| t(".fonts.#{font}", scope: 'admin_center') },
     render_affiliate_visual_design_value(@site.visual_design_json, :footer_and_results_font_family))
 
+  %fieldset
+    %legend= t('.color_scheme', scope: 'admin_center')
```

Comment:
Should this have a space on both side of the = sign?

Reply:
@MyNameIsMissing We don't do any sort of haml-linting (I may create a story for that, but suspect we have tons of violations at this point so not sure it is worth it), but this is valid [haml](https://github.com/haml/haml/blob/main/REFERENCE.md).  The percent sign denotes the html tag `<legend>`, followed immediately by an `=` which denotes that what comes next should be interpreted then printed (in this case the translation for the section heading).

This results in: `<legend>Color scheme</legend>`